### PR TITLE
ddd

### PR DESCRIPTION
**Pull request by ajax-malov-n**
DDD

### CODE REVIEW

File: domainservice/core/src/main/kotlin/systems/ajax/malov/stockanalyzer/core/infrastructure/config/MongoConfig.kt [hunk 1]
```diff
@@ -14,25 +12,28 @@ import org.springframework.data.mongodb.core.mapping.Document
...
     }
 
     @Bean
     @SuppressWarnings("ObjectLiteralToLambda")
     fun mongoCustomConversions(): MongoCustomConversions {
         val converters = listOf<Converter<*, *>>(
-            object : Converter<BigDecimal, Decimal128> {
-                override fun convert(source: BigDecimal): Decimal128 {
-                    return Decimal128(source)
+            object : Converter<BigDecimal, org.bson.types.Decimal128> {
```
Comment (ajax-torbin-v) [inline domainservice/core/src/main/kotlin/systems/ajax/malov/stockanalyzer/core/infrastructure/config/MongoConfig.kt]: Remove full package name

File: domainservice/stock-record/src/integrationTest/kotlin/util/IntegrationTestBase.kt [hunk 2]
```diff
@@ -1,22 +1,48 @@
-package systems.ajax.malov.stockanalyzer.util
+package util
```
Comment (ajax-torbin-v) [inline domainservice/stock-record/src/integrationTest/kotlin/util/IntegrationTestBase.kt]: ???

File: gateway/src/test/kotlin/arch/OnionArchitectureTest.kt [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package arch
```
Comment (ajax-shulepa-a) [inline gateway/src/test/kotlin/arch/OnionArchitectureTest.kt]: fix package name

File: gateway/src/test/kotlin/arch/OnionArchitectureTest.kt [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import com.tngtech.archunit.core.domain.JavaClasses
+import com.tngtech.archunit.core.importer.ClassFileImporter
+import com.tngtech.archunit.core.importer.ImportOption
+import com.tngtech.archunit.library.Architectures.onionArchitecture
+import org.junit.jupiter.api.BeforeAll
+import kotlin.test.Test
+
+class OnionArchitectureTest {
+
+    companion object {
```
Comment (ajax-shulepa-a) [inline gateway/src/test/kotlin/arch/OnionArchitectureTest.kt]: place companion object at the end of the file

File: settings.gradle.kts [hunk 5]
```diff
@@ -9,3 +9,11 @@ include("grpc-api")
 buildCache {
     local.isEnabled = true
 }
+include("domainservice:stock-record")
+findProject(":domainservice:stock-record")?.name = "stock-record"
```
Comment (ajax-shulepa-a) [inline settings.gradle.kts]: why do you need to rename modules?

**Review by ajax-shulepa-a — APPROVED** [review]

File: settings.gradle.kts (hunk 5 above)
Comment (ajax-malov-n) [reply settings.gradle.kts]: It was generated by IDEA. I removed each module renaming

**Pull request merged**